config/defines.inc.php

```diff
@@ -26,7 +26,7 @@
 
 /* Debug only */
 if (!defined('_PS_MODE_DEV_')) {
-    define('_PS_MODE_DEV_', true);
+    define('_PS_MODE_DEV_', false);
```

Comment:
You shouldn't push this on the repo even if it's handy for you local environment Please revert this

Reply:
Ahh on every other project this needs to be false, so I guess I accidentally changed that.